src/appforms/tests/tests/backbone/040-view02field14field_map.js

```diff
@@ -0,0 +1,163 @@
+var assert = chai.assert;
```

Comment:
Legacy of how the appforms folders are structured. 
The order of loading the files matters. 👎 

This whole section of the SDK needs to be refactored.